src/main/build/debug.c

```diff
@@ -107,4 +107,5 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "ATTITUDE",
     "VTX_MSP",
     "GPS_DOP",
+    "ANGLE_FEEDFORWARD",
```

Comment:
The debug includes not just feedforward information.  It includes angle P gain information; its focus is to tune the angle_mode of flight operation.

Maybe use `ANGLE_MODE` as in `DEBUG_MODE_ANGLE_MODE` ?

We will need a debug for Horizon mode, so if we use the above, the horizon_mode debug could be `HORIZON_MODE`.

Reply:
Please make this debug "ANGLE_MODE" so I can finalise blackbox.  I need it to be consistent with the other new modes.  I have explained why.

Reply:
OK will do